example/main.jsonnet

```diff
@@ -1,4 +1,4 @@
-local dex = (import 'github.com/observatorium/deployments/components/dex.libsonnet') + {
+local dex = (import '../components/dex.libsonnet') + {
```

Comment:
we don't expect there to be any `../components` here, right? it should be `deployments/components` or something, no?

Reply:
I really want to start pushing the absolute paths everywhere by the way. :wink: 
So going forward, if you see any, please help clean them up. :+1: